code/__DEFINES/economy.dm

```diff
@@ -32,6 +32,8 @@
 #define ACCOUNT_SEC "SEC"
 #define ACCOUNT_SEC_NAME "Defense Budget"
 
+#define ACCOUNT_CENT "CENT"
+#define ACCOUNT_CENT_NAME "Centcom Budget" //Adding this for Nanotrasen payouts (e.g. bounties) as I believe money should always come from existing financial assets (e.g. bank balances, cash)
```

Comment:
If the point is to track money more precisely, than making it all come from a magical centcom account with infinite cash is functionally no different than if you just adding the number to a balance. If it's a logging issue, than add a line to econ logging explaining that _x person_ got _y credits_ for completing a _z bounty_.

Reply:
Ultimately, I'd like to see in-game tracking and logging of this stuff and it coming out of an existing account will help with that. I'd like to see bank account statements for account holders, some sort of central finance console to see all transactions NT should be able to see and an admin overview of the system with every transaction and cause (in the case of game-generated transactions). I doubt I could figure out how to code that though. But I added this with that in mind, I guess hoping it would steer people towards something like that.

I also added it because it makes more sense to me to have the money come from somewhere feasible rather than generate it by the game directly, even if the source is functionally unlimited. At least doing it like this you can see where this payee is used in code instead of just the generic adjustment proc. I am not an accountant but this is how I believe they would do it.